addon/editor/input-handlers/delete-handler.ts

```diff
@@ -674,101 +325,102 @@ export default class DeleteHandler implements InputHandler {
     // check where our cursor is and get the deepest "thing" after
     // the cursor (character or node)
     const thingAfterCursor: ThingAfterCursor = this.getThingAfterCursor();
-    console.log(thingAfterCursor)
+    console.log(thingAfterCursor);
```

Comment:
should probably replace this with a call to `editorDebug` or remove the statement all together